_dashboards/admin-ui-index/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: default
-title: Admin UI for index operations in OpenSearch Dashboards
+title: Index management in OpenSearch Dashboards
```

Comment:
Maybe rename this to "Index and snapshot management" because snapshot management is not part of index management.